### PR TITLE
Management command to pre-seed UUID mappings table

### PR DESCRIPTION
**Pull request by gasman**

### CODE REVIEW

File: wagtail_transfer/management/commands/preseed_transfer_table.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+    def handle(self, *args, **options):
+        model_name = options['model_name'].lower()
+        try:
+            model = get_model_for_path(model_name)
+        except ObjectDoesNotExist:
+            raise ImproperlyConfigured("%r is not recognised as a model name." % model_name)
+
+        if model != get_base_model(model):
+            raise ImproperlyConfigured(
```
Comment (kaedroho) [inline wagtail_transfer/management/commands/preseed_transfer_table.py]: I think we should raise a `CommandError` here

File: wagtail_transfer/management/commands/preseed_transfer_table.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        for pk in unmapped_ids:
+            _, created = IDMapping.objects.get_or_create(
+                content_type=content_type, local_id=pk,
+                defaults={'uid': uuid.uuid5(NAMESPACE, "%s:%s" % (model_name, pk))}
+            )
+            if created:
+                created_count += 1
+
+        if options['verbosity'] >= 1:
+            print("%d ID mappings created." % created_count)
```
Comment (kaedroho) [inline wagtail_transfer/management/commands/preseed_transfer_table.py]: Use `self.stdout.write` (makes it easier to redirect output to something else in unit tests)

File: wagtail_transfer/management/commands/preseed_transfer_table.py (hunk 1 above)
Comment (gasman) [reply wagtail_transfer/management/commands/preseed_transfer_table.py]: Makes sense, thanks! Updated in 1025c3ba10788528f11e371500718dc16861cf3a

File: wagtail_transfer/management/commands/preseed_transfer_table.py (hunk 2 above)
Comment (gasman) [reply wagtail_transfer/management/commands/preseed_transfer_table.py]: Also updated in 1025c3ba10788528f11e371500718dc16861cf3a.

**Pull request merged**